src/ejabberd_c2s.erl

```diff
@@ -2536,7 +2540,8 @@ resend_subscription_requests(Acc, #state{pending_invitations = Pending} = StateD
                          A1 = send_element(A, XMLPacket, State),
                          {value, From} =  xml:get_tag_attr(<<"from">>, XMLPacket),
                          {value, To} = xml:get_tag_attr(<<"to">>, XMLPacket),
-                         BufferedStateData = buffer_out_stanza({From, To, XMLPacket}, State),
+                         PacketTuple = {jid:from_binary(From), jid:from_binary(To), XMLPacket},
```

Comment:
Do we have `from` and `to` in the Acc at this point?

Reply:
Take a look at the lines above - they are retrieved directly from `xmlel`.

Reply:
I don't know, however this code seems to be correct as it is covered with tests. I'll add this item to TODO list. :)